site/docs/uk/README.md

```diff
@@ -41,7 +41,7 @@ permalink: /uk/
 ```ts
 import { Bot } from "grammy";
 
-const bot = new Bot(""); // <-- помістіть тут маркер свого бота (https://t.me/BotFather)
+const bot = new Bot(""); // <-- помістіть тут токен свого бота (https://t.me/BotFather)
```

Comment:
I don't think it's worth using "Вказуємо" here, since we, the developers, did not specify anything, but only provide instructions

Reply:
> I don't think it's worth using "Вказуємо" here, since we, the developers, did not specify anything, but only provide instructions

Anyway, we can use "Вкажіть". What u think about this suggestion?